application/src/main/java/org/thingsboard/server/service/edge/rpc/EdgeGrpcSession.java

```diff
@@ -324,94 +318,111 @@ void processEdgeEvents() throws ExecutionException, InterruptedException {
         log.trace("[{}] processHandleMessages finished", this.sessionId);
```

Comment:
Maybe this log should also indicate whether it was executing the real work or not?